RePlays.csproj

```diff
@@ -51,13 +52,22 @@
   <PropertyGroup>
     <NugetTools>$(PkgNuGet_CommandLine)\tools</NugetTools>
     <SquirrelTools>$(PkgClowd_Squirrel)\tools</SquirrelTools>
-    <Version>1.0.0</Version>
+    <Version>1.3.1</Version>
```

Comment:
I don't think it is necessary to change the version here. Version is handled during the `dotnet publish` where we specifically pass in a version number.